tests/test_validator_version.bats

```diff
@@ -30,8 +30,8 @@ load 'test_fixtures'
     assert_output "run_validator: ${TEST_FINAL_RUN_VALIDATOR}"
 }
 
-@test "validator_version='master'" {
-    export INPUT_VALIDATOR_VERSION=master
+@test "validator_version='latest'" {
+    export INPUT_VALIDATOR_VERSION=latest
```

Comment:
This shouldn't be changed as it's a test for whether it works to install a branch.
```suggestion
@test "validator_version='master'" {
    export INPUT_VALIDATOR_VERSION=master
```